packages/dev/loaders/src/OBJ/solidParser.ts

```diff
@@ -275,6 +276,17 @@ export class SolidParser {
         //Pattern5 => triangle = ["-1/-1/-1","-2/-2/-2","-3/-3/-3","-1/-1/-1","-3/-3/-3","-4/-4/-4"];
     }
 
+    /**
+     * To get color between color and extension color
+     * @param index Integer The index of the element in the array
+     * @returns value of target color
+     */
+    private _getColor(index: number) {
+        if (this._loadingOptions.importVertexColors) {

```

Comment:
Sorry, i sometimes forget we don't have noImplicitReturns really. You will need to add `else { return; }` to make typescript happy...

or something like this:

```javascript
return  this._loadingOptions.importVertexColors ? this._extColors[index] ?? this._colors[index] : undefined;
```